variables.tf

```diff
@@ -545,6 +545,16 @@ variable "cluster_name" {
   type        = string
 }
 
+variable "cluster_type" {
+  default     = "BASIC_CLUSTER"
```

Comment:
use `basic` or `enhanced` for the user. Put the correct values when passing to the cluster resource itself.